### PR TITLE
feat: Suppress color for all hooks if `PRE_COMMIT_COLOR=never` set

### PR DESCRIPTION
**Pull request by MaxymVlasov**
Put an `x` into the box if that apply:

- [ ] This PR introduces breaking change.
- [ ] This PR fixes a bug.
- [ ] This PR adds new functionality.
- [x] This PR enhances existing functionality.

### How can we test changes

```yaml
repos:
- repo: https://github.com/antonbabenko/pre-commit-terraform
  rev: c86a8a428f59d84afb02587af3deb4bb61983351
```

```bash
PRE_COMMIT_COLOR=never pre-commit run -a
```

### CODE REVIEW

File: hooks/infracost_breakdown.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ function infracost_breakdown_ {
 
   # Get hook settings
   IFS=";" read -r -a checks <<< "$hook_config"
-
+  # Suppress infracost color
   if [ "$PRE_COMMIT_COLOR" = "never" ]; then
     args+=("--no-color")
   fi
```
Comment (MaxymVlasov) [inline hooks/infracost_breakdown.sh:41]: Also, I prefer to organize it like already done with `common::per_dir_hook` - `per_dir_hook_unique_part`, but not sure how better to do it.

File: hooks/terraform_docs.sh [hunk 2]
```diff
@@ -212,6 +229,9 @@ function terraform_docs {
 
     popd > /dev/null
   done
+
+  # Cleanup
+  rm -f "$config_file_no_color"
```
Comment (yermulnik) [inline hooks/terraform_docs.sh:234]:
```suggestion
  [ -e "$config_file_no_color" ] && rm -f "$config_file_no_color"
```

File: README.md [hunk 3]
```diff
@@ -422,7 +433,7 @@ Unlike most other hooks, this hook triggers once if there are any changed files
         * `.projects[].diff.totalHourlyCost` - show the difference in hourly cost for the existing infra and tf plan
         * `.projects[].diff.totalMonthlyCost` - show the difference in monthly cost for the existing infra and tf plan
         * `.diffTotalHourlyCost` (for Infracost version 0.9.12 or newer) or `[.projects[].diff.totalMonthlyCost | select (.!=null) | tonumber] | add` (for Infracost older than 0.9.12)
-    * To disable hook color output, set `PRE_COMMIT_COLOR=never` env var.
+
```
Comment (MaxymVlasov) [inline README.md:436]:
```suggestion
```

File: hooks/infracost_breakdown.sh (hunk 1 above)
Comment (MaxymVlasov) [reply hooks/infracost_breakdown.sh:41]: @yermulnik ?

Comment (MaxymVlasov) [reply hooks/infracost_breakdown.sh:41]: OK, maybe another PR :)

**Pull request merged**